ios-xcframework/.gitignore

```diff
@@ -13,3 +13,10 @@ fastlane/report.xml
 
 # XCFramework build process intermediate and output artifacts
 build
+
+# This is generated by the `pod install` process.
+# It is required by FBReactNativeSpec to compile in case `command -v node` does not give a result from within Xcode.
+.xcode.env
```

Comment:
[We commit](https://github.com/WordPress/gutenberg/pull/51386#discussion_r1236994369) the `.xcode.env` file in `WordPress/gutenberg` as `.xcode.env.local` is available for overriding where necessary. Should we do the same here as well?

Reply:
Yep, we could commit the file with the same content we did [here](https://github.com/WordPress/gutenberg/pull/51386#discussion_r1236994369) (i.e.`export NODE_BINARY=$(command -v node)`). This way we'd cover the same logic we are doing in the Pod file in this regard:

https://github.com/wordpress-mobile/gutenberg-mobile/blob/3bbb8b702b6a05cb2b457de83c6e31fd8795211c/ios-xcframework/Podfile#L129-L137

Reply:
I applied this suggestion in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5924/commits/bdd6ab148c810f04fab55ae620713305144a78b7.